### ISSUE TITLE
JLCPCB rejects UFC display layers

### ISSUE THREAD

**Issue by Alan-JH** [jrsteensen/OpenHornet#305]
### Summary:
JLCPCB rejected my order with display layers 2-4 for opt, com, and scratchpad, citing the minimum allowed clearance between routed cutouts being 4mm, while the PCBs have a clearance of about .32mm. 

### Expected Results/How It Should Work:
Expected JLC to process the order, and some number of pieces to fail

### Actual Results/How It Does Work:
JLC rejected the order before manufacturing.

### Screenshots:

![image](https://user-images.githubusercontent.com/49381955/118814554-46e90b00-b87e-11eb-9a15-fe16e7560b14.png)
![image](https://user-images.githubusercontent.com/49381955/118814575-4badbf00-b87e-11eb-92b9-5eedd00ebf2a.png)

## More Information
*Add an "X" in the square brackets to check the applicable checkboxs.*
### Category:
*Check one or more items.*
- [ ] MCAD (SolidWorks)
- [x] ECAD (PCB Design or other electrical hardware)
- [ ] Software - Sketch
- [ ] Software - DCS-BIOS
- [ ] Software - Library
### Type:
*Check one item.*
- [ ] Bug
- [ ] Feature Enhancement
- [x] Maintenance
- [ ] Question
- [ ] Documentation
### Applicable End Item:
*Check one item.*
- [ ] Top Level Assembly          
- [ ] Lower Instrument Panel (LIP)
- [x] Main Instrument Panel (MIP)
- [ ] Left Console
- [ ] Right Console
- [ ] Seat
- [ ] Center Tub
- [ ] Flight Stick
- [ ] Throttle
- [ ] General Software
### Associated Filename(s):

ufc_v4_comm_layer2.kicad_pcb
ufc_v4_comm_layer3.kicad_pcb
ufc_v4_comm_layer4.kicad_pcb
ufc_v4_opt_layer2.kicad_pcb
ufc_v4_opt_layer3.kicad_pcb
ufc_v4_opt_layer4.kicad_pcb
ufc_v4_scratchpad_layer2.kicad_pcb
ufc_v4_scratchpad_layer3.kicad_pcb
ufc_v4_scratchpad_layer4.kicad_pcb

**Comment by jrsteensen** [comment]
That would be great! I'm curious if clear resin will do the trick or if we will need to add a tiny bit of white powder to give the light something to reflect off of. Did you have to modify the layers to print them with a .2mm nozzle?

**Comment by Alan-JH** [comment]
It turns out the clear resin I have went bad. I could get some more soon, but for now I tried white glue instead, as I have used that to diffuse LEDs before. 

I had to close the slots for the 7 segment digits on the scratchpad by 0.05mm to get the slicer to slice the model properly with a 0.2mm nozzle. Otherwise, it didn't need modification. The nozzle clogged every 4 prints or so though, so I would either need to cold pull or replace it entirely, which wasn't pleasant, hence my suggestion for SLA.
